learning-team/2021-01/2021-01-14.md

```diff
@@ -0,0 +1,57 @@
+# January 14, 2021
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### Super Rentals Refresh (IL, 15 min)
+
+Mar, from Simplabs, has helped with (1) updating the design of and (2) improving the user experience (UX) for the [Super Rentals website](https://ember-super-rentals.netlify.app/). The proposal document can be found below. Mar has asked for our feedback with 3 UX updates.
```

Comment:
Note to Learning Team:

Before the meeting, if you can quickly go through the [Super Rentals tutorial](https://guides.emberjs.com/release/tutorial/part-1/), I think that would help discussing suggestions and potential breaking changes.